### PR TITLE
Ncsdk 10354 ml app pm

### PR DESCRIPTION
**Pull request by rakons**
Implement power management into data forwarder machine learning runner.

### CODE REVIEW

**Comment by rakons** [comment]
Waiting for #6187

**Comment by pdunaj** [comment]
@emob-nordic will set up a meeting regarding that as it seems there are some things to clarify

File: applications/machine_learning/README.rst [hunk 1]
```diff
@@ -66,6 +66,8 @@ By default, the application uses pretrained machine leaning models deployed in `
   * ``tap`` - the device is tapped while placed on a flat surface.
 
   Unknown gestures, such as shaking the device, are recognized as anomaly.
+* Thingy:53 supports sensor sleeping functionality - sensor stops sampling when there is no activity
```
Comment (divipillai) [inline applications/machine_learning/README.rst:69]:
```suggestion
* Thingy:53 supports sensor sleeping functionality - sensor stops sampling when there is no activity and resumes back when movements are detected.
```

File: applications/machine_learning/README.rst [hunk 2]
```diff
@@ -256,6 +258,11 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+  * If sensor enters into sleep mode LED would generate short blinks.
+
+    * In machine learning mode LED would blink in green color.
+    * In data forwarder it would blink in red color
```
Comment (divipillai) [inline applications/machine_learning/README.rst:264]:
```suggestion
    *  LED blink in red color in the data forwarder.
```

File: applications/machine_learning/README.rst [hunk 3]
```diff
@@ -256,6 +258,11 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+  * If sensor enters into sleep mode LED would generate short blinks.
+
+    * In machine learning mode LED would blink in green color.
```
Comment (divipillai) [inline applications/machine_learning/README.rst:263]:
```suggestion
    * LED blink in green color in the machine learning mode.
```

File: applications/machine_learning/README.rst [hunk 4]
```diff
@@ -256,6 +258,11 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+  * If sensor enters into sleep mode LED would generate short blinks.
```
Comment (divipillai) [inline applications/machine_learning/README.rst:261]:
```suggestion
  * If sensor enters into the sleep mode, the LED makes short blinks.
```

**Pull request closed without merging**